go.mod

```diff
@@ -4,7 +4,9 @@ go 1.12
 
```

Comment:
Optional: Bump minimum required Go version in go.mod? IIRC there's also some behavior change in go.mod based on version.

Reply:
I don't think this is necessary, `wire` should still work with the older versions (although we no longer promise it since we only test at latest release and latest release -1 ).